src/index.ts

```diff
@@ -9,40 +9,56 @@ export interface Paginated<T> {
   results: T[],
 }
 
+export interface Table {
+  name: string;
+  edge: boolean;
+  id: number;
+  created: string;
+  modified: string;
+  workspace: Workspace[];
```

Comment:
Why is a `Table`'s `workspace` property a *list* of `Workspace` objects? I would have thought the workspace a table belongs to to be unique.

Reply:
It's singular, so it should just be `Workspace` here, not `Workspace[]`.

Reply:
It's singular, so this should just be `Workspace` instead of `Workspace[]`.

Reply:
Sorry about that, I misread the example on the Swagger page.

See https://github.com/multinet-app/multinetjs/pull/42/commits/b3bc6b705d5dafbf44320c6a32e7bf139c4b7378#diff-a2a171449d862fe29692ce031981047d7ab755ae7f84c707aef80701b3ea0c80L18-R18